Sources/Widgets/Widgets3D/ShapeWidget/test/testShapeWidget.js

```diff
@@ -0,0 +1,74 @@
+import test from 'tape-catch';
```

Comment:
does not belong to this commit, does it ?

Reply:
No sorry, I've just removed it